css/styles.css

```diff
@@ -0,0 +1,215 @@
+:root {
+  --blanco: #f9f9f9;
+  --negro: #36383f;
+  --gris: #85888c;
+  --verde: #16d026;
+} /* variables in spanish (for fun) */
+
+body {
+  background-color: var(--blanco);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  margin: 0;
+}
+
+/* ----- Elements ----- */
+
+.index-page {
+  background-color: #fff;
+  border-radius: 0 0 0 5%;
+  background-image:
+    url("../img/piramid.png"),
+    url("../img/square.png"),
+    url("../img/revertsquare.png"),
+    url("../img/elipsir.png"),
+    url("../img/square2.png"),
+    url("../img/artrock.png");
+  background-repeat: no-repeat;
+  background-position:
+    2.5% 35%,
+    20% 74%,
+    95% 85%,
+    95% 25%,
+    80% 8%,
+    15% 10%;
+}
+
+.flex-navbar {
+  background-color: #fff;
+  color: white;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 20px;
+}
+
+.flex-nav-list {
+  list-style-type: none;
+  margin-right: 30%;
+}
+
+.flex-nav-list .list-item {
+  display: inline-block;
+  padding: 20px 10px;
+}
+
+.flex-navbar a {
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.flex-navbar a:hover {
+  color: #16d026;
+}
+
+.primary-logo {
+  height: 60px;
+}
+
+.menu-line {
+  width: 30px;
+  height: 3.5px;
+  background-color: #000;
+  margin-bottom: 5px;
+}
+
+#headline-section {
+  height: 916px;
+  margin-left: 18%;
+  margin-right: 18%;
+  margin-top: 150px;
+  background-image: none;
+}
```

Comment:
- [ ] Although you did an excellent job :+1:, please note that as a project requirement, the section needs to be aligned using flexbox. This means that you should use flexbox properties and values to align the elements within the section. The reason for this is that flexbox provides a powerful and flexible way to organize and align content within a container, making it easier to create responsive layouts and adjust to different screen sizes :+1:.
- [ ] Well done on the styling of your project :+1:. However, it is necessary that your portfolio closely resembles the Figma template as per the project requirements. Kindly be aware that the mobile version has a distinct background image that needs to be displayed from 375px. Additionally, please note that the icons are not the same. To address this, you may use different tools, and I suggest using [Font Awesome](https://fontawesome.com/icons).
- [ ] Also there are other differences listed below:
  - The title should have two different colors :+1:.
  - The size of `p` should be 20px :+1:
  - You should add the button  with the chevron down in the bottom of your page :+1:
  - Kindly check the padding and margins of your portfolio :+1:

**Your design**

![Screenshot from 2023-03-23 04-46-30](https://user-images.githubusercontent.com/105117832/227165022-37c70862-39db-4f6b-b130-afd00ef674e5.png)

**Figma**

![Screenshot from 2023-03-23 04-46-52](https://user-images.githubusercontent.com/105117832/227165111-202cc798-150a-4f72-a452-3e9b4c1ec2e2.png)